lib/locales/en.yml

```diff
@@ -22,6 +22,8 @@ en:
   eoc_composite_metadata_title: Chapter Review
   eoc_solutions_title: Solutions
   eoc_suggested_reading: Suggestions for Further Study
+  eoc_further_research_title: Further Research
+  eoc_further_reading: Further Reading
```

Comment:
Like notes, eoc sections are now being titled with `eoc.class-name` syntax. Looks like you're already doing that on [these lines](https://github.com/openstax/kitchen/pull/307/files#diff-887c311739256a99225ab8153707412184ba7b8b571ad90d45462344f25ab86cR9-R11) in the spec, but the locale should match. Also, if these titles are just used in philosophy, they might belong in the philosophy-specific locale file.